src/Sentry/Sentry.csproj

```diff
@@ -59,6 +59,14 @@
     </Compile>
   </ItemGroup>
 
+  <!--
+    We use TestableIO to be able to mock the FileSystem during tests
+    https://github.com/TestableIO/System.IO.Abstractions
+  -->
+  <ItemGroup>
+    <PackageReference Include="TestableIO.System.IO.Abstractions.TestingHelpers" Version="20.0.4" PrivateAssets="all" />
```

Comment:
See Notes in description. We're not surfacing this to the public API but use it exclusively internaly.
Helps us with keeping a clean cache during tests due to mocking a filesystem in memory.